projects/go-lib/src/lib/components/go-side-nav/_nav-variables.scss

```diff
@@ -0,0 +1,4 @@
+$inner-font-size: 0.833rem;
```

Comment:
I don't know how I feel about all of these variables being in a separate file.
1. Are there any variables in gosheets we could use for this?
2. We should probably pull these out into a more global settings file.